<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Preferences</class>
 <widget class="QDialog" name="Preferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>591</width>
     <height>321</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>General</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>551</width>
          <height>80</height>
         </rect>
        </property>
        <property name="title">
         <string>System</string>
        </property>
        <widget class="QCheckBox" name="desktopNotifications">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>541</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>Show Desktop Notifications</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QCheckBox" name="startOnStartup">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>541</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>Start OSF Offline on Computer Startup</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>110</y>
          <width>561</width>
          <height>61</height>
         </rect>
        </property>
        <property name="title">
         <string>Choose Folder to Place Project in </string>
        </property>
        <widget class="QPushButton" name="changeFolderButton">
         <property name="geometry">
          <rect>
           <x>440</x>
           <y>20</y>
           <width>99</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Change</string>
         </property>
        </widget>
        <widget class="QTextEdit" name="containingFolderTextEdit">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>20</y>
           <width>331</width>
           <height>31</height>
          </rect>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;/home/himanshu/somefolder/My Project&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>OSF (logged in)</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>581</width>
          <height>51</height>
         </rect>
        </property>
        <property name="title">
         <string>Account</string>
        </property>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="geometry">
          <rect>
           <x>280</x>
           <y>50</y>
           <width>561</width>
           <height>51</height>
          </rect>
         </property>
         <property name="title">
          <string>Account</string>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>16</y>
           <width>411</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>User name</string>
         </property>
        </widget>
        <widget class="QPushButton" name="accountLogOutButton">
         <property name="geometry">
          <rect>
           <x>460</x>
           <y>10</y>
           <width>99</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Log Out</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>561</width>
          <height>211</height>
         </rect>
        </property>
        <property name="title">
         <string>Project</string>
        </property>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>541</width>
           <height>81</height>
          </rect>
         </property>
         <property name="title">
          <string>Choose Project to Sync With</string>
         </property>
         <widget class="QRadioButton" name="radioButton">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>20</y>
            <width>117</width>
            <height>22</height>
           </rect>
          </property>
          <property name="text">
           <string>Project A</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="radioButton_2">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>40</y>
            <width>117</width>
            <height>22</height>
           </rect>
          </property>
          <property name="text">
           <string>Project B</string>
          </property>
         </widget>
         <widget class="QPushButton" name="changeFolderButton_2">
          <property name="geometry">
           <rect>
            <x>440</x>
            <y>20</y>
            <width>99</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Change</string>
          </property>
         </widget>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>OSF (logged out)</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_7">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>40</y>
          <width>441</width>
          <height>181</height>
         </rect>
        </property>
        <property name="title">
         <string>Log in</string>
        </property>
        <widget class="QPushButton" name="logInButton">
         <property name="geometry">
          <rect>
           <x>330</x>
           <y>150</y>
           <width>99</width>
           <height>27</height>
          </rect>
         </property>
         <property name="text">
          <string>Log In</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="emailEdit">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>30</y>
           <width>261</width>
           <height>27</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="passwordEdit">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>70</y>
           <width>261</width>
           <height>27</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>26</y>
           <width>61</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Email</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>70</y>
           <width>81</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Password</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Priority Syncing</string>
       </attribute>
       <widget class="QTreeWidget" name="treeWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>561</width>
          <height>221</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="acceptDrops">
         <bool>false</bool>
        </property>
        <property name="sortingEnabled">
         <bool>false</bool>
        </property>
        <property name="animated">
         <bool>false</bool>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="headerHidden">
         <bool>false</bool>
        </property>
        <attribute name="headerCascadingSectionResizes">
         <bool>true</bool>
        </attribute>
        <attribute name="headerDefaultSectionSize">
         <number>200</number>
        </attribute>
        <attribute name="headerHighlightSections">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Component, Folder, or File</string>
         </property>
         <property name="textAlignment">
          <set>AlignJustify|AlignVCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Priority</string>
         </property>
         <property name="textAlignment">
          <set>AlignJustify|AlignVCenter</set>
         </property>
        </column>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>561</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Priority Items will be given priority when syncing</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>About</string>
       </attribute>
       <widget class="QTextEdit" name="textEdit_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>551</width>
          <height>251</height>
         </rect>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;This is copyright info. This is copyright info. This is copyright info. This is copyright info. This is copyright info. This is copyright info. This is copyright info. This is copyright info. This is copyright info. This is copyright info. This is copyright info. This is copyright info. This is copyright info. This is copyright info. This is copyright info. This is copyright info. This is copyright info. This is copyright info. This is copyright info. This is copyright info. This is copyright info. This is copyright info. This is copyright info. This is copyright info. This is copyright info. This is copyright info. This is copyright info. This is copyright info. This is copyright info. This is copyright info. This is copyright info. This is copyright info. This is copyright info. This is copyright info. This is copyright info. This is copyright info. This is copyright info. This is copyright info. This is copyright info. This is copyright info. This is copyright info. This is copyright info. This is copyright info. This is copyright info. This is copyright info. This is copyright info. This is copyright info. This is copyright info. This is copyright info. This is copyright info. This is copyright info. This is copyright info. This is copyright info. This is copyright info. This is copyright info. This is copyright info. This is copyright info. &lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Â© Center for Open Science&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
